lwcell/src/system/lwcell_ll_stm32.c

```diff
@@ -94,9 +96,9 @@ usart_ll_thread(void* arg) {
 
         /* Read data */
 #if defined(LWCELL_USART_DMA_RX_STREAM)
-        pos = sizeof(usart_mem) - LL_DMA_GetDataLength(LWCELL_USART_DMA, LWCELL_USART_DMA_RX_STREAM);
+        pos = sizeof(usart_mem) - LL_DMA_GetBlkDataLength(LWCELL_USART_DMA, LWCELL_USART_DMA_RX_STREAM);
```

Comment:
lwcell_ll_stm32.c is a generic file, aiming to support all STM32 series. So if BlkDataLength is used, it will only work on U5 and nowhere else.